### PR TITLE
Patch 1

### PR DESCRIPTION
**Pull request by BjornMelgaard**
https://github.com/char0n/ramda-adjunct/issues/24#issuecomment-342761872

### CODE REVIEW

File: src/flattenPath.js [hunk 1]
```diff
@@ -2,7 +2,7 @@ import { pathOr, curry, merge } from 'ramda';
 
 /**
  * Flattens a property path so that its fields are spread out into the provided object.
- *
+ * It's like {@link RA.spreadPath|spreadPath}, but preserves object under property path
```
Comment (char0n) [inline src/flattenPath.js]: `.` at the end of the sentence

File: src/flattenPath.js [hunk 2]
```diff
@@ -14,7 +14,7 @@ import { pathOr, curry, merge } from 'ramda';
  * @param {!Array.<string|number>} path The property path to flatten
  * @param {!Object} obj The provided object
  * @return {!Object} The flattened object
- * @see {@link RA.flattenProp|flattenProp}
+ * @see {@link RA.flattenProp|flattenProp} {@link RA.spreadPath|spreadPath}
```
Comment (char0n) [inline src/flattenPath.js]: `{@link RA.flattenProp|flattenProp}, {@link RA.spreadPath|spreadPath}` - use comma as delimiter

File: src/flattenProp.js [hunk 3]
```diff
@@ -4,6 +4,7 @@ import flattenPath from './flattenPath';
 
 /**
  * Flattens a property so that its fields are spread out into the provided object.
+ * It's like {@link RA.spreadProp|spreadProp}, but preserves object under property path
```
Comment (char0n) [inline src/flattenProp.js]: dot

File: src/flattenProp.js [hunk 4]
```diff
@@ -15,7 +16,7 @@ import flattenPath from './flattenPath';
  * @param {!string|number} prop The property to flatten
  * @param {!Object} obj The provided object
  * @return {!Object} The flattened object
- * @see {@link RA.flattenPath|flattenPath}
+ * @see {@link RA.flattenPath|flattenPath} {@link RA.spreadProp|spreadProp}
```
Comment (char0n) [inline src/flattenProp.js]: comma

File: src/spreadPath.js [hunk 5]
```diff
@@ -2,6 +2,7 @@ import { curryN, converge, merge, dissocPath, pathOr } from 'ramda';
 
 /**
  * Spreads object under property path onto provided object.
+ * It's like {@link RA.flattenPath|flattenPath}, but removes object under property path
```
Comment (char0n) [inline src/spreadPath.js]: dot

File: src/spreadPath.js [hunk 6]
```diff
@@ -13,13 +14,13 @@ import { curryN, converge, merge, dissocPath, pathOr } from 'ramda';
  * @param {!Array.<string|number>} path The property path to spread
  * @param {!Object} obj The provided object
  * @return {!Object} The result of the spread
- * @see {@link RA.spreadProp|spreadProp}
+ * @see {@link RA.spreadProp|spreadProp} {@link RA.flattenPath|flattenPath}
```
Comment (char0n) [inline src/spreadPath.js]: comma

File: src/spreadProp.js [hunk 7]
```diff
@@ -4,6 +4,7 @@ import spreadPath from './spreadPath';
 
 /**
  * Spreads object under property onto provided object.
+ * It's like {@link RA.flattenProp|flattenProp}, but removes object under property
```
Comment (char0n) [inline src/spreadProp.js]: dot

File: src/spreadProp.js [hunk 8]
```diff
@@ -15,7 +16,7 @@ import spreadPath from './spreadPath';
  * @param {!string|number} prop The property to spread
  * @param {!Object} obj The provided object
  * @return {!Object} The result of the spread
- * @see {@link RA.spreadPath|spreadPath}
+ * @see {@link RA.spreadPath|spreadPath} {@link RA.flattenProp|flattenProp}
```
Comment (char0n) [inline src/spreadProp.js]: comma

File: src/flattenPath.js (hunk 1 above)
Comment (char0n) [reply src/flattenPath.js]: In all sentences use `under the property`;  **the**

**Comment by BjornMelgaard** [comment]
done

**Pull request merged**